censys/asm/inventory.py

```diff
@@ -31,17 +31,29 @@ def search(
         Returns:
             dict: Inventory search results.
         """
+        if workspaces is not None:
+            print(
```

Comment:
I think we have a deprecation log type.

Reply:
I see the `DeprecationDecorator` but I wasn't sure how to use that as a log type?